tests/test_events_default.json

```diff
@@ -1,10 +1,10 @@
 {
-  "births": 602,
-  "deaths": 339,
-  "recoveries": 13609,
-  "contacts": 94331,
-  "wanings": 12116,
-  "reassortments": 21,
+  "births": 606,
+  "deaths": 321,
+  "recoveries": 30943,
+  "contacts": 215625,
+  "wanings": 23578,
+  "reassortments": 76,
```

Comment:
what's the reason these changed? was it the `stable=True` changes the results even on linux?